src/dependency-registry.ts

```diff
@@ -416,13 +454,13 @@ export function handleSupportedCoreContractAdded(
     return;
   }
 
-  coreContract.dependencyRegistry = event.address;
+  coreContract.latestDepenencyRegistryOverrideAllowlistedOn = event.address;
```

Comment:
```suggestion
  coreContract.latestDependencyRegistryOverrideAllowlistedOn = event.address;
```

Reply:
fixed in [a981a0a](https://github.com/ArtBlocks/artblocks-subgraph/pull/328/commits/a981a0a44af5112297a3b25e5e34a708b237c261)